lib/ansible/modules/cloud/docker/docker_swarm_service_info.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+#
+# (c) 2019 Hannes Ljungberg <hannes.ljungberg@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: docker_swarm_service_info
+
+short_description: Retrieves information about docker services from a Swarm Manager
+
+description:
+  - Retrieves information about a docker service.
+  - Essentially returns the output of C(docker service inspect <name>).
+  - Must be executed on a host running as Swarm Manager, otherwise the module will fail.
+
+version_added: "2.8"
+
+options:
+  name:
+    description:
+      - The name of the service to inspect.
+    type: str
+extends_documentation_fragment:
+  - docker
+  - docker.docker_py_1_documentation
+
+author:
+  - Hannes Ljungberg (@hannseman)
+
+requirements:
+  - "L(Docker SDK for Python,https://docker-py.readthedocs.io/en/stable/) >= 2.0.0"
+  - "Docker API >= 1.24"
+'''
+
+EXAMPLES = '''
+- name: Get info from a service
+  docker_swarm_service_info:
+    name: myservice
+  register: result
+'''
+
+RETURN = '''
+exists:
+    description:
+      - Returns whether the service exists.
+    type: bool
+    returned: always
+    sample: true
+service:
+    description:
+      - A dictionary representing the current state of the service. Matches the C(docker service inspect) output.
+      - Will be C(None) if service does not exist.
```

Comment:
```suggestion
      - Will be C(none) if service does not exist.
```
That thing is called `none` in jinja2. (`xxx is none` works, while `xxx is None` does not -- at least when I last tested that.)

Reply:
You're right. Always confuses me since `False` and `True` works.

Reply:
It used to confuse me too, but I think I now got used to it after a (long) time ;)